CHANGELOG-18.0.md

```diff
@@ -36,4 +36,174 @@ You can see all the changes done to the package that you carry about with this t
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
-<!-- Add generated changelog below this line -->
+<!-- Release notes generated using configuration in .github/release.yml at 18.0 -->
+
+## What's Changed
+
+### :sparkles: Enhancements and features
+
+- added multiselect component by @chlebektomas in https://github.com/shopsys/shopsys/pull/4152
+- updated form validation by @chlebektomas in https://github.com/shopsys/shopsys/pull/4181
+- added better accessibility for banner slider by @chlebektomas in https://github.com/shopsys/shopsys/pull/4158
+- Introduced factories for all implementations of DataSourceInterface + QueryBuilderDataSources now order textual columns with correct collation by @TomasLudvik in https://github.com/shopsys/shopsys/pull/4135
+- Add special article page (about) GTM type by @JanMolcik in https://github.com/shopsys/shopsys/pull/4184
```

Comment:
nitpick 🤓
I would unify the capitalization of the first letter in all bullet points.